configure

```diff
@@ -1,4 +1,14 @@
-#!/usr/bin/env python
+#!/bin/sh
+
+# Locate python2 interpreter and re-execute the script.  Note that the
+# mix of single and double quotes is intentional, as is the fact that
+# the ] goes on a new line.
+_=[ 'exec' '/bin/sh' '-c' '''
+which python2.7 >/dev/null && exec python2.7 "$0" "$@"
+which python2 >/dev/null && exec python2 "$0" "$@"
```

Comment:
So, I deliberately didn't add python2.6 for two reasons:

1. I wasn't sure if it should go before or after python2, and
2. We supported python2.6 because of centos 5 but since we no longer support that platform...